### PR TITLE
Pylons appears

### PR DESCRIPTION
**Pull request by MikeSofaer**
Using DNS, is that OK? Probably good for DDOS.

### CODE REVIEW

File: contestant_info/pylons.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "chain_id":"pylonschain",
+  "node addr":"tcp://node.pylons.tech:26657"
```
Comment (ajcronk) [inline contestant_info/pylons.json]: Missing underscore node_addr?

Comment (MikeSofaer) [reply contestant_info/pylons.json]: Should be fixed

**Comment by jessysaurusrex** [comment]
Hi @MikeSofaer! Confirming receipt of your RPC endpoint -- once your team whitelists the IP addresses from the confirmation email, you are all set for Game of Zones. 🚀 

(And about the underscore, we broke the json 🙈, we will get to it!)

**Comment by MikeSofaer** [comment]
Hello! Thanks! I think everything is pretty wide open, but I’m working on starting up a relayer to make sure I can see it from my machine at home.

**Pull request merged**